### PR TITLE
Add a Dev Servers category with http-server and live-server

### LINKED ISSUE #2: add live-server
Not sure in which category to put it, but [live-server](https://github.com/tapio/live-server) is easily in my top 5 npm scripts utilities. Typically used to develop minimalist static pages lacking their own live-reloading servers like Github Pages. [example](https://github.com/inventaire/stack/blob/master/package.json#L7)

### PR DESCRIPTION
**Pull request by maxlath**
fixed #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - [Articles](#articles)
 - [Task Runners](#task-runners)
 - [File Watchers](#file-watchers)
+- [Dev Servers](#doc-servers)
```
Comment (RyanZim) [inline README.md]: Typo here, use `npm run doctoc` to automatically & correctly generate a TOC.

File: README.md [hunk 2]
```diff
@@ -43,6 +44,11 @@ Tools for running multiple commands or npm scripts in parallel or sequentially.
 - [onchange](https://github.com/Qard/onchange) - `onchange <glob> -- <command>`.
 - [watch](https://github.com/mikeal/watch) - `watch <command> <directory>`.
 
+## Dev Servers
+
+- [http-server](https://github.com/indexzero/http-server) - A simple zero-configuration command-line http server
+- [live-server](https://github.com/tapio/live-server) - Simple development http server with live reload capability
```
Comment (RyanZim) [inline README.md]: Please end lines with a full stop (`.`).

**Review by RyanZim — CHANGES_REQUESTED** [review]
Couple of nitpicks, other than that, LGTM. Thanks!

File: README.md (hunk 1 above)
Comment (maxlath) [reply README.md]: I tried to run use `npm run doctoc` but for a reason I can't find, it always come back to the state before my edit, even after having manually edited the ToC oO

Comment (RyanZim) [reply README.md]: Did you save your changes to the file first?

**Review by RyanZim — APPROVED** [review]

**Comment by RyanZim** [comment]
@maxlath Confirmed the buggy doctoc behavior, issue filed: https://github.com/thlorenz/doctoc/issues/121.

**Pull request merged**